deploy/firesim

```diff
@@ -15,8 +15,8 @@ from datetime import date
 import warnings
 from cryptography.utils import CryptographyDeprecationWarning
 with warnings.catch_warnings():
-    if date.today() < date(2022, 8, 31):
-        # get rid of deprecation warnings until 2022-08-31
+    if date.today() < date(2022, 9, 15):
```

Comment:
@abejgonzalez suggested we talk about this in today's meeting. Two weeks is not very far out...